samples/subsys/usb/webusb/src/main.c

```diff
@@ -24,10 +24,10 @@ static volatile bool data_transmitted;
 static volatile bool data_arrived;
 static u8_t data_buf[64];
 
-/* WebUSB Platform Capability Descriptor */
+/* WebUSB + MS Platform Capability Descriptors */
 static const u8_t webusb_bos_descriptor[] = {
 	/* Binary Object Store descriptor */
-	0x05, 0x0F, 0x1D, 0x00, 0x01,
+	0x05, 0x0F, 0x39, 0x00, 0x02,
```

Comment:
This *will* create a merge conflict with #7094. :cry:

Reply:
I was asking to make a separate patch, you can still merge those two branches

Reply:
If there is a PR already then just move that path to be the first in this series.

Reply:
OK, this calls for a bit of clever rebasing.